recipes/wget/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: wget
+  version: '1.18'
+
+about:
+  home: http://www.gnu.org/software/wget/
+  license: GPLv3
+  summary: 'Retrieve files using HTTP, HTTPS and FTP'
+
+source:
+  fn: wget-1.18.tar.xz
+  url: http://ftp.gnu.org/gnu/wget/wget-1.18.tar.xz
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python >=3
```

Comment:
Python as a build dependency? The source does look like pure C to me... Are you sure about this?